src/components/cylc/tree/TreeItem.vue

```diff
@@ -281,6 +334,50 @@ export default {
       classes['node-data'] = true
       classes[`node-data-${this.node.type}`] = true
       return classes
+    },
+    /**
+     * Get the summary of custom messages for a node.
+     * @param {{
+     *   node: {
+     *     taskProxy: {
+     *       prerequisites: [
+     *         {
+     *           conditions: [
```

Comment:
Alongside conditions you may want to get the expression (as sometimes there are aliases for conditional statements)

Reply:
Not sure how/when I would use it though?

![image](https://user-images.githubusercontent.com/304786/107899870-700aae80-6fa4-11eb-8b7e-e57aaef683ba.png)

Is it so that we can link the prerequisite with the condition?